.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
+- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
+- [ ] Gallery example in `./doc/examples` (new features only)
+- [ ] Unit tests
+
+[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]
+
+## References
```

Comment:
I bet we need to elaborate here a bit more.
@stefanv please take a look at the refactoring of `References` section in the `ISSUE_TEMPLATE` (moved its content to the `Description` desction). Probably, we should make here the same change.
I like your proposal from the first sight, but it doesn't take into account 'enhancement' category, as well as 'optional' nature of the requested information (I'd agree with @JDWarner that 'If known' is really one of the best wordings here).
To sum up, looking forward for your and 1 more feedback on moving this into the `Description`, and preference on implicit/explicit version. Thanks!